entrypoint.sh

```diff
@@ -80,4 +80,6 @@ if [[ "$DIFF" != "" ]];then
     echo "${DIFF}"
 fi
 
-/action/main.js "${DIFF}"
+# This variable is deliberately unquoted so that it can be properly processed on the other side.
+# shellcheck disable=SC2086
+/action/main.js ${DIFF}
```

Comment:
That is sooooo odd; normal rule of thumb is to ALWAYS quote arguments in shell scripts. Would love to know why this occurs...

Reply:
Same. I think the list of files gets read as a single string, but I'm not certain. It should be easy enough to see what's happening around https://github.com/laminas/laminas-ci-matrix-action/blob/5e50256611e353e2f4f002caf1d99a631889e97f/src/main.ts#L22 though.